### PR TITLE
test 

### PR DESCRIPTION
**Pull request by DamienVStone**
comment

### CODE REVIEW

File: API/Tenders.API.Integration.JobSeeker/Program.cs [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-using System.Threading.Tasks;
-using Tenders.API.Integration.Interfaces;
-using Tenders.API.Integration.JobSeeker.Interfaces;
-using Tenders.Core.DI;
-using Tenders.Core.Abstraction.Models.Enums;
-using Tenders.Core.Abstraction.Services;
-using Tenders.Core.Models;
+﻿// <copyright file="Program.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
```
Comment (DamienVStone) [inline API/Tenders.API.Integration.JobSeeker/Program.cs]: It's ok

File: API/Tenders.API.Integration.JobSeeker/Program.cs [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+    using Tenders.API.Integration.Interfaces;
+    using Tenders.API.Integration.JobSeeker.Interfaces;
+    using Tenders.Core.Abstraction.Models.Enums;
+    using Tenders.Core.Abstraction.Services;
+    using Tenders.Core.DI;
+    using Tenders.Core.Models;
+
     internal class Program
     {
         private static IApiActionsService apiDataProvider;
```
Comment (DamienVStone) [inline API/Tenders.API.Integration.JobSeeker/Program.cs]: good naming

File: Tenders.sln [hunk 3]
```diff
@@ -75,7 +75,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tenders.RTS", "RTS\Tenders.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tenders.RTS.Tests", "RTS\Tenders.RTS.Tests\Tenders.RTS.Tests.csproj", "{D871BDE0-6104-4446-BDDB-E7B1D8281B12}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Tenders.RTS.Requesting", "RTS\Tenders.RTS.Requesting\Tenders.RTS.Requesting.csproj", "{1C0DCB6D-59A2-4122-8617-EBE2ED9A18A7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tenders.RTS.Requesting", "RTS\Tenders.RTS.Requesting\Tenders.RTS.Requesting.csproj", "{1C0DCB6D-59A2-4122-8617-EBE2ED9A18A7}"
+EndProject
```
Comment (DamienVStone) [inline Tenders.sln]: test review comment

**Comment by ssds-team** [comment]
test declined

**Pull request closed without merging**